docs/microservices.json

```diff
@@ -1659,28 +1659,145 @@
             "id": "sscs-tya-notif",
             "name": "SSCS Track Your Appeal Nofiication",
             "group": "SSCS",
-            "proType": "api",
+            "type": "api",
             "description": "",
-            "repository": "https://github.com/hmcts/track-your-appeal-notifications",
+            "repository": "https://github.com/hmcts/sscs-track-your-appeal-notifications",
             "spec": "https://hmcts.github.io/reform-api-docs/specs/track-your-appeal-notifications.json",
             "dependencies": [
                 {
                     "id": "ccd-data-store-api",
                     "hard": true,
                     "apis": []
                 },
+                {
+                    "id": "dm-store",
+                    "hard": true,
+                    "apis": []
+                },
                 {
                     "id": "rpe-service-auth-provider",
                     "hard": true,
                     "apis": []
                 },
                 {
-                    "id": "sscs-tya-frontend",
+                    "id": "cmc-pdf-service",
```

Comment:
this has finally converted to `rpe-pdf-service` but that's not part of this pr - needs a global update